include/deal.II/lac/matrix_block.h

```diff
@@ -132,6 +132,12 @@ class MatrixBlock : public Subscriptor
    */
   MatrixBlock(const MatrixBlock<MatrixType> &M);
 
+  /**
+   * Assignment operator.
+   */
+  MatrixBlock<MatrixType> &
+  operator=(const MatrixBlock<MatrixType> &) = default;
```

Comment:
I had to look up if the copy constructor does anything special here in order to just if defaulting the copy assignment operator is OK. Both of them are, so I would just also default the other operator while being here.